sys/include/net/gnrc/ipv6/nib/nc.h

```diff
@@ -233,6 +233,32 @@ static inline unsigned gnrc_ipv6_nib_nc_get_ar_state(const gnrc_ipv6_nib_nc_t *e
 int gnrc_ipv6_nib_nc_set(const ipv6_addr_t *ipv6, unsigned iface,
                          const uint8_t *l2addr, size_t l2addr_len);
 
+#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_6LN) || defined(DOXYGEN)
+/**
+ * @brief   Adds an unmanaged neighbor entry to the NIB if the interface represents a 6LN node
+ *          and the IPv6 address can be constructed from the L2 address @p l2addr.
+ *
+ * @param[in] iface         The interface to the neighbor.
+ * @param[in] l2addr        The neighbor's layer 2 address.
+ * @param[in] l2addr_len    Length of @p l2addr.
+ *
+ * @return  0 on success.
+ * @return  -ENOTSUP, if the interface does not represent a 6LN or when
+ *           gnrc_netif_t::device_type of the iface does not support IID conversion.
+ * @return  -EINVAL, when @p addr_len is invalid for the
+ *          gnrc_netif_t::device_type of @p netif.
+ * @return  -ENOMEM, if no space is left in neighbor cache.
+ */
+int gnrc_ipv6_nib_nc_set_6ln(unsigned iface, const uint8_t *l2addr,
+                                 size_t l2addr_len);
+#else /* CONFIG_GNRC_IPV6_NIB_6LN */
+static inline int gnrc_ipv6_nib_nc_set_6ln(unsigned iface, const uint8_t *l2addr,
+                                 size_t l2addr_len)
+{
+   return -ENOTSUP;
```

Comment:
```suggestion
    (void)iface;
    (void)l2addr;
    (void)l2addr_len;

    return -ENOTSUP;
```

To fix the `unused parameter` errors when the function is not supported.
Also the indentation level was incorrect (three spaces instead of four).